src/raft.rs

```diff
@@ -1238,7 +1238,7 @@ impl<T: Storage> Raft<T> {
             m.commit = commit;
             m.commit_term = commit_term;
             if campaign_type == CAMPAIGN_TRANSFER {
-                m.context = campaign_type.to_vec();
+                m.context = campaign_type.to_vec().into();
```

Comment:
Perhaps we can add a new method without allocation. `Bytes::from_static_bytes` will just store the static pointer.